src/components/MdxComponents/fileDownloadCard.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { useTheme } from 'emotion-theming';
+import { Download } from 'react-feather';
+import emoji from '../../utils/emoji';
+import { decreaseIntensivity } from '../../utils/colors';
+import Card from './card';
+
+const DownloadCard = styled(Card)`
+  cursor: pointer;
+  flex-direction: row;
+  align-items: center;
+  &:hover {
+    border: 1px solid ${(props) => props.theme.colors.primary};
+  }
+`;
+
+const DownloadPath = styled.div`
+  color: ${(props) => decreaseIntensivity(props.theme.colors.fontLight, 0.25)};
+  font-size: 9pt;
+  padding-left: 16px;
+  text-align: right;
+`;
+
+const Title = styled.div`
+  padding: 0 14px;
+  color: ${(props) => props.theme.colors.primary};
+  font-size: 12pt;
+  font-weight: 500;
+  flex: 1;
+`;
+
+export default ({ title, url }) => {
```

Comment:
Codacy found an issue: [Component definition is missing display name](https://app.codacy.com/manual/matfilipowicz/BooGi/pullRequest?prid=5999112)